pubsub/google/cloud/pubsub_v1/publisher/batch/base.py

```diff
@@ -114,7 +114,7 @@ def will_accept(self, message):
 
         # If this message will make the batch exceed the ``max_bytes``
         # setting, return False.
-        if self.size + message.ByteSize() > self.settings.max_bytes:
+        if self.size + message.ByteSize() >= self.settings.max_bytes:
```

Comment:
`>` was correct.

Reply:
Ah yes. In my brain, the check was

```python
self.size > self.settings.max_bytes
```

DERP